peru/main.py

```diff
@@ -221,6 +221,31 @@ def do_override(params):
             print('{}: {}'.format(module, params.runtime.get_override(module)))
 
 
+@peru_command('module', '''\
+Usage:
+    peru module <list> [-z]
+
+Options:
+    -z              print one module per line for machine-readable formatting
+
+''')
+def do_list(params):
+    modules = params.scope.modules.values()
+    output = ''
+    delim = ' | ' if params.args['-z'] else "\n  "
+
+    for module in modules:
+        output += module.name
+        for field, val in module.plugin_fields.items():
+            output += '{}{}: {}'.format(delim, field, val)
+        if(params.runtime.get_override(module.name)):
+            output += '{}{}: {}'.format(delim, 'override', params.runtime.get_override(module.name))
+        output += '\n'
```

Comment:
Unfortunately we need to worry about plugin fields whose values have newlines in them. For example, although I don't think git allows newlines in branch/tag/whatever names, one of the other VCS's we support (in the future?) might.